powerdnsadmin/models/user.py

```diff
@@ -317,26 +330,22 @@ def is_validate(self, method, src_ip='', trust_user=False):
                                         .format(self.username))
                                     return False
 
-                                for group in [
-                                        g.decode("utf-8")
-                                        for g in user_ad_member_of
-                                ]:
-                                    user_ldap_groups += self.ad_recursive_groups(
-                                        group)
+                                for i in  range(len(user_ad_member_of)):
```

Comment:
Can you keep the comprehension list ?
```suggestion
                                user_ad_member_of = [g.decode("utf-8") for g in user_ad_member_of]
```